platform/solr/solr-factory/src/main/java/org/codice/solr/factory/impl/HttpSolrClientFactory.java

```diff
@@ -281,7 +284,9 @@ private static SSLContext getSslContext() {
         | KeyStoreException
         | KeyManagementException e) {
       throw new IllegalArgumentException(
-          "Unable to use javax.net.ssl.keyStorePassword to load key material to create SSL context for Solr client.");
+          "Unable to use "
+              + KEY_STORE_PASS
```

Comment:
✏️ This particular change seems a little silly to me. I don't know how I feel about breaking out an exception message to insert a const in the middle of it.